_html/templates/my-profile-page.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+* My profile page - view mode.
+*/
+
+include "inc/header.inc.php";
+include "modules/header.php";
+?>
+
+<div class="section-intro sponsors-intro">
+    <div class="section-intro--background"></div>
+</div>
+
+<div class="profile-wrapper">
```

Comment:
We had a rule in which the name of the module is the same everywhere. In this case it would be something like my-profile.php. The in SCSS we would have my-profile.scss and the top class in the mark-up (here, instead of profile-wrapper) would be also .my-profile. The wrapper could be added then inside, like my-profile--wrapper.